sample-apps/communication/main.js

```diff
@@ -45,3 +56,52 @@ function onNavigateUsingMessageClick () {
   }
   veraApi.sendArmeMessage({ packageName: navigationPackageName, data })
 }
+
+async function onNearestPoiClick() {
+  const MAX_DISTANCE_V = 2
+  const MAX_DISTANCE_H = 3
+  let cameraCoords = veraApi.getCameraPose().translation
+  const filter = {
+    bool: {
+      filter: [{
+        range: {
+          "ar:geometry.center_y": {
+            gte: cameraCoords[1] - MAX_DISTANCE_V,
+            lte: cameraCoords[1] + MAX_DISTANCE_V
+          }
+        }
+      }]
+    }
+  }
+  // Geo-filter array: camX, camZ, max 2D horizontal distance (m)
```

Comment:
I think `camX` and `camZ` are too specific, the terminology should be something like centerX centerZ to clarify you can also issue this search centered at any point in space - not necessarily around the user/camera

Reply:
Agreed, added this comment to a sequencial PR, where it is relevat